src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+    private String color;
```

Comment:
String color - is a common field for all Figures. Move it to Figure class please